src/components/storagePools/storagePool.jsx

```diff
@@ -206,10 +205,6 @@ class StoragePoolActions extends React.Component {
                           isOpen={this.state.isActionOpen}
                           position='right'
                           dropdownItems={dropdownItems} />
-                {this.state.showDeleteModal &&
-                <StoragePoolDelete close={() => this.setState({ showDeleteModal: false })}
-                                   storagePool={storagePool}
-                                   vms={vms} />}
```

Comment:
`StoragePoolActions` component still has `showDeleteModal` property initialized in its state. It should be removed from there too.

Reply:
Yes, thansk! Fixed.